meson.build

```
@@ -132,6 +132,8 @@ endif
 subdir('include')
 subdir('src')
 subdir('tools')
-subdir('tests')
-subdir('fuzz')
+if get_option('skip-tests') == false
```

```suggestion
if not get_option('skip-tests')
```

However, as I notice the original commit actually made the tests option be true by default -- it would run the tests by default, which is behavior preserving, and to disable the tests you actually have to run with `-Dtests=false`

- `-Dtests=false`
- `-Dskip-tests=true`

Semantically, these mean the same thing, and I'm inclined to think the first one reads a bit more naturally. So I'd probably just drop the second commit in the end.